### PR TITLE
Unify integration doc updates 2

### PR DESCRIPTION
**Pull request by jieatelement**
*Issue #, if available:*
NA

*Description of changes:*
Remove redundant Element Unify description from architecture, deployment steps, and product description sections.
Add a link to the workload template to launch Unify with existing VPC.
Add default values to S3 and key prefix parameters in both main and workload templates.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/partner_editable/deploy_steps.adoc [hunk 1]
```diff
@@ -37,9 +37,7 @@ Each deployment takes about {deployment_time} to complete.
...
-+
-Element Unify uses a no code/low code approach to model building using metadata from input sources such as PTC KepServerEX, Ignition from Inductive Automation, P&IDs, and PI Asset Framework from AVEVA, and feed models to AWS IoT SiteWise. Unify is suitable for single site deployments and for organizations looking to establish a solid foundation for their operational data infrastructure with a view to roll out across multiple regions and sites.
+.. *Option 2 - Element Unify* Element Unify uses a no code/low code approach to model building using metadata from input sources such as PTC KepServerEX, Ignition from Inductive Automation, P&IDs, and PI Asset Framework from AVEVA, and feed models to AWS IoT SiteWise. Unify is suitable for single site deployments and for organizations looking to establish a solid foundation for their operational data infrastructure with a view to roll out across multiple regions and sites.
```
Comment (sshvans) [inline docs/partner_editable/deploy_steps.adoc:40]: Please remove this verbiage from here. Deployment steps should only focus on deployment. This text is already convered in other place and its just talking about the benefit of element unify and not the deployment. You can just say something like "Element unify can be deployed in following 2 ways:"

File: docs/partner_editable/deploy_steps.adoc [hunk 2]
```diff
@@ -37,9 +37,7 @@ Each deployment takes about {deployment_time} to complete.
...
 +
 Fully integrated with the AWS Integrated Machine Connectivity (IMC) kit, Unify enables you to:
 +
```
Comment (sshvans) [inline docs/partner_editable/deploy_steps.adoc:43]: Remove the bullet points and links to the external pages from here as well.

**Review by sshvans — CHANGES_REQUESTED** [review]

File: docs/partner_editable/deploy_steps.adoc (hunk 2 above)
Comment (jieatelement) [reply docs/partner_editable/deploy_steps.adoc]: Thanks @sshvans. I've cut down the content and left it as one sentence aligned with Option 1.

Comment (sshvans) [reply docs/partner_editable/deploy_steps.adoc]: Thanks! We also need to remove the below bullet points.

**Review by sshvans — APPROVED** [review]

File: docs/partner_editable/deploy_steps.adoc (hunk 2 above)
Comment (sshvans) [reply docs/partner_editable/deploy_steps.adoc]: Never mind, I think you have alreaady taken care of that I believe.

**Pull request merged**